sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/query.sql

```diff
@@ -178,6 +186,7 @@ fxa_subscriptions AS (
     ) AS pricing_plan,
     -- Stripe default billing grace period is 1 day and Paypal is billed by Stripe
     INTERVAL 1 DAY AS billing_grace_period,
+    IF(promotion_code IS NULL, [], [promotion_code]) AS promotion_codes,
```

Comment:
changed from `[promotion_code]` to this, because arrays can't contain null elements